docs/mathematics/Matrix-Addition.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Matrix Addition
+slug: /mathematics/Matrix-Addition
+---
+
+## Introduction
+
+In mathematics, matrix addition is the operation of adding two matrices by adding the corresponding entries together. Two matrices must have an equal number of rows and columns to be added.
+
+## Explanation
+
+Consider two matrices A and B , the sum of two matrices A and B will be a matrix which has the same number of rows and columns as A and B. The sum of A and B, denoted A + B, It is computed by adding corresponding elements of A and B
```

Comment:
Can you make a diagram using some prototyping tool and add them here?

Reply:
I tried to insert an image using markdown syntax but it is not working here

Reply:
Just use `![image](IMAGE_URL)`

Reply:
Sorry @HarshCasper It was an issue with the image url